pkg/k8shandler/visualization.go

```diff
@@ -432,32 +436,75 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaConsoleExternal
 		return err
 	}
 
-	consoleExternalLogLink := NewConsoleExternalLogLink(
-		"kibana",
+	consoleExternalLogLinkForAdmin := NewConsoleExternalLogLink(
+		"kibana-admin",
+		cluster.Namespace,
+		"Show in Kibana",
+		strings.Join([]string{kibanaURL,
+			"/app/kibana#/discover?_g=(time:(from:now-1w,mode:relative,to:now))&_a=(columns:!(kubernetes.container_name,message),index:",
+			".operations.*",
```

Comment:
not all records in the operations index will have a pod, namespace, and container name... it may be logs that came from the node. do we really want to filter those out immediately?

Reply:
I assume this link is displayed (thus also used) only when you list pods in web console and click logs tab. So in this context all logs should have all these filters applied.
Shall we address other user cases as well?